source/JiraIntegration.sln.DotSettings

```diff
@@ -1,3 +1,253 @@
 ﻿<wpf:ResourceDictionary xml:space="preserve" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:s="clr-namespace:System;assembly=mscorlib" xmlns:ss="urn:shemas-jetbrains-com:settings-storage-xaml" xmlns:wpf="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
+	<s:Boolean x:Key="/Default/CodeInspection/CodeAnnotations/NamespacesWithAnnotations/=Octopus_002ETools_002EAnnotations/@EntryIndexedValue">True</s:Boolean>
```

Comment:
Was this on purpose?

Reply:
It was at the time, the formatting was wrong. I'll drop this out of here though and do a separate PR with this and all the resulting format updates.